code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,448 @@
+		to_chat(owner, span_notice("You scratch furiously at [bodypart] to ruin the cloth that hides the beauty!"))
+
+
+
+// Climb over the rusted mountain, gives a brain trauma causing the person to randomly rust tiles beneath them
+/obj/item/wallframe/painting/eldritch/rust
+	name = "Climb over the rusted mountain"
+	desc = "A painting depicting something climbing a mountain of rust, it gives you an eerie feeling."
+	icon_state = "eldritch_painting_rust"
+	result_path = /obj/structure/sign/painting/eldritch/rust
+
+/obj/structure/sign/painting/eldritch/rust
+	name = "Climb over the rusted mountain"
+	desc = "A painting depicting something climbing a mountain of rust, it gives you an eerie feeling. Destroyable with wirecutters."
+	icon_state = "eldritch_painting_rust"
+	sensor_type = /datum/proximity_monitor/advanced/eldritch_painting/rust
+
+// Moodlets used to prevent rust and give a positive moodlet for heretics
+/datum/mood_event/eldritch_painting/rust_examine
+	description = "The rusted climb can wait..."
+	mood_change = -2
+	timeout = 3 MINUTES
+
+/datum/mood_event/eldritch_painting/rust_heretic_examine
+	description = "I must hurry the rusted climb!"
+	mood_change = 6
+	timeout = 5 MINUTES
+
+// The special examine interaction for this painting
+/obj/structure/sign/painting/eldritch/rust/examine(mob/living/carbon/human/user)
+	..()
+	if("rusted_examine" in user.mob_mood.mood_events)
+		return
+
+	if(IS_HERETIC(user))
+		to_chat(user, "You see the climber, and are inspired by it!")
+		user.add_mood_event("rusted_examine", /datum/mood_event/eldritch_painting/rust_heretic_examine)
+
+	if(user.has_trauma_type(/datum/brain_trauma/severe/rusting))
+		to_chat(user, "It can wait...")
+		user.add_mood_event("rusted_examine", /datum/mood_event/eldritch_painting/rust_examine)
+
+
+// Specific proximity monitor for Climb over the rusted mountain or /obj/item/wallframe/painting/eldritch/rust
+/datum/proximity_monitor/advanced/eldritch_painting/rust
+	applied_trauma = /datum/brain_trauma/severe/rusting
+	text_to_display = "It climbs, and I will aid it...The rust calls and I shall answer..."
+
+/*
+ * A brain trauma that this eldritch paintings apply
+ * This one is for "Climb over the rusted mountain" or /obj/structure/sign/painting/eldritch/rust
+ */
+/datum/brain_trauma/severe/rusting
+	name = "The Rusted Climb"
+	desc = "Patient seems to oxidise things around them at random, and seem to believe they are aiding a creature in climbing a mountin."
+	scan_desc = "C_)L(#_I_##M;B"
+	gain_text = span_warning("The rusted climb shall finish at the peak")
+	lose_text = span_notice("The rusted climb? Whats that? An odd dream to be sure.")
+	random_gain = FALSE
+
+/datum/brain_trauma/severe/rusting/on_life(seconds_per_tick, times_fired)
+	var/atom/tile = get_turf(owner)
+	// If they have the special mood event for rusted climb they don't start randomly rusting tiles beneath them
+	if("rusted_examine" in owner.mob_mood.mood_events)
```

Comment:
please do not use mood events as a trait system. should add a trait and check for it here

Reply:
Working on it

Reply:
Should be addressed in latest commit.